sites/all/themes/ocha_basic/sass/styles.scss

```diff
@@ -40,7 +40,6 @@
 //@import "hrinfo-custom/global/font";
 @import "hrinfo-custom/contacts";
 @import "hrinfo-custom/form";
-@import "hrinfo-custom/highcharts";
```

Comment:
@left23 Are you sure this is not needed ? Because normally, Highcharts is used for FTS graphs...

Reply:
Yep, I added this initially. It wasn't part of the original theme.
It's purpose was to override a rule from ocha_basic in sites/all/themes/ocha_basic/sass/cd/_cd-icons.scss which hides all SVG in content.
I've since change that rule because it was too strong, so no override is needed.
See diff:
https://github.com/humanitarianresponse/site/pull/789/commits/b1e45cf10e1cb7727e00636eee317ef14b75e038#diff-9f0484420615feaf89dd53c7e9bc4856L6